examples/redux-saga-typescript-graphql-fluentui/README.md

```diff
@@ -0,0 +1,43 @@
+# redux-saga-typescript-graphQL-fluentUI example
+
+It is a simple example of how to build next app by using redux, redux-saga, typesript, GraphQL and UI libary FluentUI. you will have to build your own GraphQl endpoint to work with the UI, the UI looks like
```

Comment:
> you will have to build your own GraphQl endpoint to work with the UI

The example has to work locally either by just running `npm run dev` (ideally), or provide detailed instructions for how to get it running.